docs/topics/js/js-to-kotlin-interop.md

```diff
@@ -100,9 +100,10 @@ external fun newC()
 
 ### @JsExport annotation
 
-> The `@JsExport` annotation is currently marked as experimental. Its design may change in future versions.
+> This feature is [Experimental](components-stability.md#stability-levels-explained).
+> Its design may change in future versions.
```

Comment:
It would be worth it to keep as much consistency as possible in the sentences explaining the experimental flag here and in @JsStatic.

Reply:
Yeah, that's what I tried to achieve: changing a note to a warning and adding the link. The problem is that this is not a new feature and that the description we agreed upon with the team back then.